README.md

```diff
@@ -16,6 +16,52 @@ bumped following the rules below:
 * New features (such as CSI spec bumps with no breaking changes) will be released as a `MINOR` update.
 * Significant breaking changes makes a `MAJOR` update.
 
+## Features
+
+Below is a list of functionality implemented by the plugin. In general, [CSI features](https://kubernetes-csi.github.io/docs/features.html) implementing a specific aspect of the [specification](https://github.com/container-storage-interface/spec/blob/master/spec.md) are available on any DigitalOcean Kubernetes version for which beta support for the feature is provided.
+
+See also the [project examples](/examples/kubernetes) for applicable use cases.
```

Comment:
How is an "applicable use case" different from an ordinary run of the mill "use case"?